### PR TITLE
add task solution

### PR DESCRIPTION
**Pull request by twisterpilot**
- [DEMO LINK](https://twisterpilot.github.io/layout_search-bar-airbnb/)
- [TEST REPORT LINK](https://twisterpilot.github.io/layout_search-bar-airbnb/report/html_report/)

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+  --linear-gradient-focus: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  --font-color: #3d4e61;
+  --font-size-big: 16px;
+  --font-size-small: 14px;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-style: normal;
```
Comment (loralevitska) [inline src/style.css:14]: Remove redundant default value
```suggestion
```

File: src/style.css [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 800;
+}
+
+body {
+  font-family: Avenir, sans-serif;
+}
+/* #endregion */
+/* #region main */
```
Comment (loralevitska) [inline src/style.css:29]: Remove comments. Try it write obvious code that explain all we need without comments)

**Review by loralevitska — APPROVED** [review]
Good job! Check my comments for improvement

File: src/style.css (hunk 2 above)
Comment (twisterpilot) [reply src/style.css:28]: Noted! As for the comments, I just liked how Misha used sections in layout_landing-page task :)

**Pull request closed without merging**